samples/timer_wakeup/src/main.c

```diff
@@ -162,9 +162,6 @@ void main(void)
 	npmx_led_mode_set(npmx_led_get(npmx_inst, LED_IDX), NPMX_LED_MODE_HOST);
 	npmx_led_state_set(npmx_led_get(npmx_inst, LED_IDX), false);
 
-	/* Save the compare value data to internal timer's registers. */
-	npmx_timer_task_trigger(m_timer_inst, NPMX_TIMER_TASK_STROBE);
-
```

Comment:
Shouldn't call to `npmx_timer_config_set` be also moved below `npmx_led_state_set`. Now, led config is set after the timer config is applied, not before, like it used to be.

Reply:
I checked and the sample is working as before.